packages/ipfs-daemon/src/ipfs-daemon.ts

```diff
@@ -148,9 +142,8 @@ export class IpfsDaemon {
                     ...configuration.ipfsEnableApi && {API: `/ip4/${configuration.tcpHost}/tcp/${configuration.ipfsApiPort}`},
                     ...configuration.ipfsEnableGateway && {Gateway: `/ip4/${configuration.tcpHost}/tcp/${configuration.ipfsGatewayPort}`}
                     ,
+                    Announce: configuration.announceAddressList,
```

Comment:
Why is this passed to both `config` and `libp2p.config`?

Reply:
I was actually wondering the same thing, it shouldn't be necessary to pass the list in both places.